scripts/run_performance_job.py

```diff
@@ -530,6 +531,16 @@ def run_performance_job(args: RunPerformanceJobArgs):
 
     runtime_type = ""
 
+    if android_mono:
+        runtime_type = "Mono"
+        configurations["CompilationMode"] = "JIT"
```

Comment:
Thinking about it more, should we set the `CompilationMode` here? I wonder if it wouldn't be better set in `runtime-perf-jobs` or even better pass it from runtime [build-perf-sample-apps.yml](https://github.com/dotnet/runtime/pull/112939/files/7537ddaf6356607da7a52b50b938305229d7cf34#diff-3dc60775dc330784b4779475969eba1cf329b336c637a230988b7dd493cb7923).

Setting it here looks like it would block extending the testing to more configurations in the future or am I missing something?

Reply:
There is definitely some room to play around with exactly how the CompilationMode gets set. I think getting a better idea of exactly which concepts fit in the same category vs being an individual thing would help, such as is JIT and R2R mutually exclusive would be what is in compilationmode? 

For now, especially because this is how we are currently setting CompilationMode in other cases, I think we should stick with this, and come up with a better design once I more fully understand the different configuration options and how they fit together.